shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/Shiva.scala

```diff
@@ -287,7 +302,12 @@ class Shiva(val parent: Configurable)
 
       annotation.foreach { toucan =>
         toucan.outputDir = new File(outputDir, "annotation")
-        toucan.inputVcf = vc.finalFile
+        toucan.inputVcf = vc.finalFile.get
+        add(toucan)
+      }
+      annotationForSomaticVariants.foreach { toucan =>
+        toucan.outputDir = new File(outputDir, "annotation")
```

Comment:
Are you sure about this? The name of the output file from Toucan is decided based on the input file, so the file names for output from 'annotation' and 'annotationForSomaticVariants' will differ.